### PR TITLE
JE-168: Added fields to user entity

### PR DESCRIPTION
**Pull request by martinyde**

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -33,6 +33,36 @@ class User extends BaseUser
      */
     private $portalApps = [];
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $UserFullName;
```
Comment (rimi-itk) [inline src/Entity/User.php]: Properties should be written in camelCase.
There is no need to prefix properties in the `User` class with `User` – we already now that they are `User` properties.
This property should be called `$fullName` and the same goes for the rest of the new properties.

File: src/Form/PortalAccountType.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        HammerService $hammerService
+    ) {
+        parent::__construct($choiceListFactory);
+        $this->hammerService = $hammerService;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $options['choice_loader'] = new CallbackChoiceLoader(function () {
+            // Note: We assume that all roles are reachable from ROLE_ADMIN.
```
Comment (rimi-itk) [inline src/Form/PortalAccountType.php]: Aha! Remember to remove/update comments when stealing code …

File: config/packages/easy_admin.yaml [hunk 3]
```diff
@@ -33,6 +33,18 @@ easy_admin:
                       label: portal.user.portalApps
                     - property: lastLogin
                       label: portal.user.lastLogin
+                    - property: UserFullName
```
Comment (rimi-itk) [inline config/packages/easy_admin.yaml]: See other comment on property names.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
In general this look very good, but the `User` properties should be renamed.

File: config/packages/easy_admin.yaml [hunk 4]
```diff
@@ -62,18 +50,18 @@ easy_admin:
...
-                      label: portal.user.UserDepartment
-                    - property: UserAddress
-                      label: portal.user.UserAddress
-                    - property: UserPostalCode
-                      label: portal.user.UserPostalCode
-                    - property: UserCity
-                      label: portal.user.UserCity
-                    - property: UserAccount
-                      label: portal.user.UserAccount
+                    - property: FullName
```
Comment (rimi-itk) [inline config/packages/easy_admin.yaml]: The property names should be camelCased. And should the labels.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
We need more camelCase!

File: src/Entity/User.php (hunk 1 above)
Comment (martinyde) [reply src/Entity/User.php]: fixed camelCake

File: config/packages/easy_admin.yaml (hunk 3 above)
Comment (martinyde) [reply config/packages/easy_admin.yaml]: fixed camelCake

File: src/Form/PortalAccountType.php (hunk 2 above)
Comment (martinyde) [reply src/Form/PortalAccountType.php]: Removed!

File: config/packages/easy_admin.yaml (hunk 4 above)
Comment (martinyde) [reply config/packages/easy_admin.yaml]: fixed camelCake

**Review by rimi-itk — APPROVED** [review]
Godkendt!

**Pull request merged**